src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -391,6 +391,17 @@ def is_equal_cell(self, other) -> bool:
             return False
         return True
 
+    def get_chemical_symbols(self) -> list:
```

Comment:
I think that the chemical symbols are always strings, so the output type signature can be `list[str]` even